lib/galaxy/objectstore/s3.py

```diff
@@ -23,14 +23,18 @@
 
 from .s3_multipart_upload import multipart_upload
 from ..objectstore import convert_bytes, ObjectStore
+from cloudbridge.cloud.factory import CloudProviderFactory, ProviderList
 
+# TODO: Cloudbridge is not exposing exceptions; however, it is planned in the next release milestone.
```

Comment:
We track issues via GitHub. I understand the versatility of a code-related TODO as a pointer to a technical debt, however many of the TODOs added in this PR would be imho better served as actual GitHub issues.